eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Mgmt/src/Properties/launchSettings.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
why is this removed?

Reply:
this is not removed. This is moved to another directory with a lot of changes in its content, therefore github does not recognize it as a rename.

Reply:
looks like, `Management` is a bad word - somehow this file (after moving) is ignored by gitignore.
I am investigating.

Reply:
It turns out launchSettings will be ignored by default.
Maybe here we just need to force add this file into git.